.github/workflows/ci.yaml

```diff
@@ -5,15 +5,20 @@ on:
     branches:
       - "v2"
 
-env: 
+env:
```

Comment:
ci.yaml has changed name to terraform.yaml on v2, shouldn't it be changed here as well?